pages/linux/true.md

```diff
@@ -0,0 +1,7 @@
+# true
+
+> Does nothing, but return a successful exit status code.
```

Comment:
Perhaps ...`but returns a successful exit status code of 0`... would read better?